lib/WTSI/DNAP/Warehouse/Schema/Result/LighthouseSample.pm

```diff
@@ -63,10 +63,16 @@ Id for this sample provided by the Lighthouse lab
 =head2 cog_uk_id
 
   data_type: 'varchar'
-  is_nullable: 1
+  is_nullable: 0
   size: 255
 
```

Comment:
```suggestion
Consortium-wide id, generated by Sanger on import to LIMS
```
Not worth to keep the comment from the previous version of the field?

Reply:
I do not think we can, unless we consistently reject updates to this table.